src/NexusMods.Games.RedEngine/Cyberpunk2077/Diagnostics.cs

```diff
@@ -263,7 +267,8 @@ Modding Cyberpunk 2077 on Linux requires you to configure WINE or the game will
         .WithSeverity(DiagnosticSeverity.Critical)
         .WithSummary("Install required packages to mod on Linux")
         .WithDetails("""
-Modding Cyberpunk 2077 on Linux requires packages to be installed in your WINE prefix.
+Modding Cyberpunk 2077 on Linux requires certain packages to be installed in your WINE prefix. Winetricks packages are often required for modding because they provide essential components or libraries that the game or its mods depend on to function correctly. 
```

Comment:
It's the app that requires Winetricks to be able to manage the game installation correctly, rather than the game or the mods.